common/core/events.c

```diff
@@ -92,41 +92,41 @@
 /*****************************************************************************
  * GLOBAL FUNCTIONS
  *****************************************************************************/
-void get_events(evt_config_t evt_config, evt_status_t *p_evt_status) {
-  if (p_evt_status == NULL) {
-    return;
-  }
+evt_status_t get_events(uint8_t event_config, uint32_t timeout) {
+  evt_status_t status = {0};
 
-  /* Configure event getters if required */
```

Comment:
Why remove comment?

Reply:
seems it is not necessary to keep the comment.